modules/fts/pages/fts-creating-indexes.adoc

```diff
@@ -1,5 +1,6 @@
 [#Creating-Indexes]
 = Creating Indexes
+:page-aliases: fts-creating-index.adoc
```

Comment:
@RayOffiah But there are no other references to fts-creating-index.adoc.

Prior to this change I found references to fts-creating-index.adoc and fts-creating-indexes.adoc. I've fixed one of them as part of this commit.

Reply:
Yes, but having the page-aliases also means that if a user has bookmarked the old page name, then they'll be redirected to the new page name.